src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -321,6 +321,99 @@ def validate_select(namespace):
 
 # region COMMAND VALIDATORS
 
+
+def process_download_batch_parameters(namespace):
+    """Process the parameters for storage blob download command"""
+    from azure.cli.command_modules.storage.storage_url_helpers import parse_url
+
+    # 1. quick check
+    if namespace.destination is None:
```

Comment:
If destination/source are required parameters and not supplied, this will never be reached. Argparse will throw an error first.